testplans/lotus-soup/deals_e2e.go

```diff
@@ -72,10 +72,13 @@ func dealsE2E(t *testkit.TestEnvironment) error {
 		}
 	}
 
-	time.Sleep(12 * time.Second)
+	// give some time to the miner, otherwise, we get errors like:
+	// deal errored deal failed: (State=26) error calling node: publishing deal: GasEstimateMessageGas
+	// error: estimating gas used: message execution failed: exit 19, reason: failed to lock balance: failed to lock client funds: not enough balance to lock for addr t0102: escrow balance 0 < locked 0 + required 640297000 (RetCode=19)
+	time.Sleep(50 * time.Second)
```

Comment:
Ideally the miner would be able to publish an event when it's ready to start accepting deals.